data/nautilus/open-tilix.py

```diff
@@ -128,14 +128,23 @@ def get_background_items(self, window, file_):
         items = []
         if file_.get_uri_scheme() in REMOTE_URI_SCHEME:
             item = Nautilus.MenuItem(name='NautilusPython::open_bg_remote_item',
-                                     label=_(u'Open Remote Tilix Here'),
-                                     tip=_(u'Open Remote Tilix In This Directory'))
+                                     label=_(u'Open in Remote Tilix'),
+                                     tip=_(u'Open Remote Tilix in this directory'))
             item.connect('activate', self._menu_activate_cb, file_)
             items.append(item)
 
-        item = Nautilus.MenuItem(name='NautilusPython::open_bg_file_item',
-                                 label=_(u'Open Tilix Here'),
-                                 tip=_(u'Open Tilix In This Directory'))
-        item.connect('activate', self._menu_background_activate_cb, file_)
-        items.append(item)
+            item = Nautilus.MenuItem(name='NautilusPython::open_bg_file_item',
```

Comment:
I don't see why this is needed here. This is only shown for ftp/sftp mounted folders. There's no "local" thing here at all.